homeassistant/components/zha/__init__.py

```diff
@@ -74,6 +74,25 @@
     extra=vol.ALLOW_EXTRA,
 )
 
+PLATFORMS = (
+    Platform.ALARM_CONTROL_PANEL,
+    Platform.BINARY_SENSOR,
+    Platform.BUTTON,
+    Platform.CLIMATE,
+    Platform.COVER,
+    Platform.DEVICE_TRACKER,
+    Platform.FAN,
+    Platform.LIGHT,
+    Platform.LOCK,
+    Platform.NUMBER,
+    Platform.SELECT,
+    Platform.SENSOR,
+    Platform.SIREN,
+    Platform.SWITCH,
+    Platform.UPDATE,
+)
+
+
 # Zigbee definitions
 CENTICELSIUS = "C-100"
```

Comment:
Unchanged in this PR, but I don't see `CENTICELSIUS` being used anywhere (before or after).
(can also be removed in a subsequent PR if we want to keep the diff "small" 😄, just noticed it now)
```suggestion
```